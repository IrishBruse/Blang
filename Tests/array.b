main() {
    extrn printf, malloc;

    auto p;
    p = malloc(16);

    printf("%p", p);
}
