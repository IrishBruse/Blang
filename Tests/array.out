System.InvalidOperationException: Cannot start process because a file name has not been provided.
   at System.Diagnostics.Process.Start()
   at System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   at BLang.Utility.Executable.Capture(String executable, String arguments) in /home/econn/git/Blang/BLang/Utility/Executable.cs:line 27